rdt/hyper_transformer.py

```diff
@@ -80,73 +78,62 @@ class HyperTransformer:
         'b': 'boolean',
         'M': 'datetime',
     }
-
-    def __init__(self, transformers=None, copy=True, dtypes=None, dtype_transformers=None):
-        self.transformers = transformers
-        self._transformers = {}
+    _DEFAULT_OUTPUT_TYPES = [
+        'numerical',
+        'float',
+        'int'
```

Comment:
The final name is not fully decided, but I suppose we will end up calling it `integer` rather than `int`.